README.md

```diff
@@ -22,9 +22,10 @@ _how it works:_
 `standard-version` does the following:
 
 1. bumps the version in _package.json_ (based on your commit history)
-2. uses [conventional-changelog](https://github.com/conventional-changelog/conventional-changelog) to update _CHANGELOG.md_
-3. commits _package.json_ and _CHANGELOG.md_
-4. tags a new release
+2. bumps the _version_ and _android-versionCode_ in _config.xml_ (based on your commit history)
```

Comment:
let's keep the original copy here, and as suggested above add a section on Cordova (we can do the same if we add more technologies).